### ISSUE TITLE
Does this work with 5.x?

### ISSUE THREAD

**Issue by herbdool** [aghstrategies/com.aghstrategies.civimonitor#10]
@agh1 does this extension work with 5.x? I guess we could test it out but just wondering if the info.xml could be updated if so.

**Comment by MegaphoneJon** [comment]
@herbdool The core functionality of this extension was brought into core Civi with version 4.7 - the same API that powers the System Status page can be queried to get all the info this extension picks up and more.  See #7.  Andrew and I each have our own version of the Nagios check we prefer - I forget what the differences are though!  My [check_civicrm](https://github.com/MegaphoneJon/check_civicrm) script could certainly use a README though...I'll do a minimal one now.

**Comment by MegaphoneJon** [comment]
Also, especially relevant to you - my version of `check_civicrm` has Backdrop support :)

**Comment by agh1** [comment]
Yeah, I really should edit this project to steer people in the right direction.  Don't use this extension: as @MegaphoneJon says, the system status check in 4.7/5.x makes it redundant.

Depending upon your monitoring infrastructure, there are two ways you can run the status check: Jon's updated [check_civicrm](https://github.com/MegaphoneJon/check_civicrm) uses the REST API to connect to sites, which is nice for having all checks executed centrally.  I recently wrote a new [check_civicrm_cv](https://github.com/aghstrategies/check_civicrm_cv) check that uses cv, Drush, or WP-CLI to execute the check locally, which works nicely on distributed infrastructure (and doesn't require keeping track of site and API keys for all your sites).  In the readme for the latter, I made a little table comparing the two.

**Comment by herbdool** [comment]
Thank you both!
